### DISCUSSION TITLE
My host meta has a wrong url, how can i regenerate it with the correct one?

### DISCUSSION THREAD

**Discussion by scerelli** [mastodon/mastodon#28512] [Q&A]
I recently set up my own Mastodon instance. After some initial struggles, I managed to get everything working using Docker. However, there's one thing I don't understand: the URL in the host-meta file within .well-known is incorrect. Rather than pointing to the subdomain of my mastodon instance, it points to the top-level domain. How could this happen? Is there a way to regenerate or correct this?

you can check it here: 
`http://mastodon.pigro.me/.well-known/host-meta`

**Comment by vmstan** [comment]
What is the setting for `LOCAL_DOMAIN` in your env file?
